src/remediations/controller.fifi.js

```diff
@@ -58,6 +58,11 @@ exports.connection_status = errors.async(async function (req, res) {
 });
 
 exports.executePlaybookRuns = errors.async(async function (req, res) {
+    //--------------------------------------------------
+    // get remediation by id
+    // get connection status of referenced systems
+    // createPlaybookRun
+    //--------------------------------------------------
```

Comment:
I'm a big fan of comments and this definitely makes the function clearer!  Thanks for adding this!